AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/Support/MockIdentity.swift

```diff
@@ -31,12 +31,32 @@ struct MockIdentity: CognitoIdentityBehavior {
         self.getCredentialsCallback = getCredentialsCallback
     }
 
+    @available(*, renamed: "getId(input:)")
     func getId(input: GetIdInput, completion: @escaping (SdkResult<GetIdOutputResponse, GetIdOutputError>) -> Void) {
         getIdCallback?(input, completion)
     }
+    
+    func getId(input: GetIdInput) async throws -> GetIdOutputResponse {
```

Comment:
Should these changes be part of this PR?

Reply:
Added these changes because of build failures when running the unit tests. I guess these were missed after migrating to `aws-sdk-swift`